packages.windows/bootstrap/extra/dcos_internal_utils/bootstrap.py

```diff
@@ -0,0 +1,122 @@
+import logging
```

Comment:
This file is almost completely duplicated from the one at `packages/bootstrap/extra/dcos_internal_utils/bootstrap.py`. We need to reuse this code rather than copy it.